cheddar/cheddar-application/src/main/java/com/clicktravel/cheddar/application/retry/RetryableUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2014 Click Travel Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.clicktravel.cheddar.application.retry;
+
+public class RetryableUtils {
+
+    private static boolean retryableDisabled = false;
+
+    public static boolean isRetryableDisabled() {
```

Comment:
It is less confusing if this were `isRetryableEnabled`, and likewise with the `boolean` field